test/parallel/test-crypto-sign-verify.js

```diff
@@ -390,13 +390,19 @@ assert.throws(
     verify.update(new clazz());
   });
 
-  [1, {}, [], Infinity].forEach((input) => {
+  [{}, [], 1, Infinity].forEach((input) => {
```

Comment:
why change this line? I think to keep the minimal changes is good

Reply:
Sorry, that was a leftover. Fixed.